### ISSUE TITLE
Molar Mass: Conflicting with Periodic table IA (Suggestion) 

### ISSUE THREAD

**Issue by adityatandon007** [duckduckgo/zeroclickinfo-goodies#4240]
**Issue:**
We already have a related IA [periodic table](https://duck.co/ia/view/periodic_table) and some of the triggers of this IA conflicts with that IA 

For ex: Screenshots:
![screenshot from 2017-06-02 17-45-03](https://cloud.githubusercontent.com/assets/25108385/26725660/3e86697a-47bd-11e7-8dfe-4b0605b5d39d.png) 

![screenshot from 2017-06-02 17-46-22](https://cloud.githubusercontent.com/assets/25108385/26726035/096c2764-47bf-11e7-9811-edae87362d25.png)

~~There is **one more slight issue** this IA has: It doesn't display the tab on the top.~~ (**Fixed**)

~~![imageedit_1_7716006707](https://cloud.githubusercontent.com/assets/25108385/26725745/9461d8c0-47bd-11e7-8cce-df7201af4d54.png)~~

People to notify:
@joshliu @pjhampton @moollaza 

**Suggestion**: Reduce some of the triggers like `atomic mass` and `atomic weight`. Let this IA trigger only when user types `molar mass <element|compound>` so it doesn't conflict with the periodic table IA.

**P.S.** With all due respect to the hard work done by @joshliu this IA can be merged with the periodic table IA or vice versa at any later stage.

------
IA Page:  http://duck.co/ia/view/molar_mass 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @joshliu

**Comment by joshliu** [comment]
Hmm this is strange, the tab is displayed in the developer environment. Am I forgetting some boilerplate code or something? Also for sure, I'll make a new pr to change the triggers once #4238 is merged.

![image](https://cloud.githubusercontent.com/assets/4336885/26739956/cab6dbda-47a1-11e7-98c8-c74b7cce86a5.png)

**Comment by pjhampton** [comment]
Yeah, it's a meta data issue I'm about to look into.
As for priority, do you guys not think this should get the priority? It seems more flexible and will get more coverage?

**Comment by joshliu** [comment]
I think because "atomic weight/mass" usually refers to the mass of a single atom (i.e. element), while "molar mass" refers to the mass of a mole of a compound, I think its more appropriate to let "atomic weight/mass" trigger the Periodic Table IA instead.

On another note, I also think that it would be a good idea to add a trigger allowing the compound to come first, so that we can accept queries such as "nacl molar mass" or "sodium chloride molar mass".

**Comment by pjhampton** [comment]
TBH, I'm no expert on this topic, so I will let you take the lead @joshliu 👍

**Comment by pjhampton** [comment]
It looks like the meta data is fixed now 😄  
https://duckduckgo.com/?q=molar+mass+of+h2o&ia=molarmass

**Comment by adityatandon007** [comment]
Yeah :smile: the tab is issue is fixed now. 
> I think because "atomic weight/mass" usually refers to the mass of a single atom (i.e. element), while "molar mass" refers to the mass of a mole of a compound, I think its more appropriate to let "atomic weight/mass" trigger the Periodic Table IA instead

I agree with @joshliu on this
